src/styles/main.scss

```
@@ -1,3 +1,63 @@
+$date-size: 100px;
+$gap-size: 1px;
+$month-length: 31;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: ($date-size + $gap-size) * 7 - 1;
+
+  &__day {
+    position: relative;
+    width: $date-size;
+    height: $date-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    line-height: $date-size;
+    text-align: center;
+    transition: all 0.5s;
```

```suggestion
    transition: transform 0.5s,
      background-color 0.5s;
```
adding all is bad practise and impact on performance